src/applications/personalization/profile/components/account-security/AccountSecurityContent.jsx

```diff
@@ -89,9 +89,12 @@ export const AccountSecurityContent = ({
             security on VA.gov.
           </p>
 
-          <h4>Go to FAQs about these topics:</h4>
+          <h4 className="vads-u-margin-top--1">
```

Comment:
Are these margin overrides really required? If they don't agree with the mocks I'd think that whatever our design system does by default is the better source of what's correct.

Reply:
The reason for these overrides is mostly the spacing between the `a` tags, which without an explicit margin will render awkwardly close to one another. So I definitely need to keep those ones 🙂 

I agree that it's generally better practice to not override `h` tag styles like this, I removed the added styling and the screenshot below reflects how it looks now

![image](https://user-images.githubusercontent.com/14869324/98565941-0e2f0000-2263-11eb-9321-36c1efcd48af.png)